src/components/header/header.js

```diff
@@ -0,0 +1,52 @@
+import React from 'react';
+
+
+export default class Header extends React.Component {
+    constructor () {
+        super()
+
+        this.state = {
+            text : ' '
+        };
+
+        this.pressEnter = this.pressEnter.bind(this);
+        this.onChangeValue = this.onChangeValue.bind(this);
+    }
+
+    onChangeValue(e) {
+        this.setState({
+            text: e.target.value
+        })}
+
+
+   pressEnter (e) {
```

Comment:
if you define a class member via arrow function, you don't need explicit binding for the methods

```javascript
pressEnter = e => {
     ....
}
```